tests/test_graphical_units/test_graph.py

```diff
@@ -7,17 +7,18 @@
 
 class PlotFunctions(GraphScene):
     CONFIG = {
-        "x_min": -10,
-        "x_max": 10.3,
-        "y_min": -1.5,
-        "y_max": 1.5,
```

Comment:
Question: These aren't set using `config`, yet have been removed from `CONFIG`. Is this a general cleanup step?

Reply:
No, they were causing problems because of multiple inheritance. (see last commit). Basically, one of its parent classes was setting these in __init__, but this class was setting it in CONFIG, and manim did not like that at all.

The current fix to pass the tests is very hacky but it will all go away when CONFIG goes away...